test/plots/piePlotTests.ts

```diff
@@ -412,14 +348,43 @@ describe("Plots", () => {
         svg.remove();
```

Comment:
Line 341, message should be balanced with the previous `it()`. i.e. mention that retrieves "correct selections for the provided dataset" or something to reflect that this time we pass some argument to the `selection()` method